packages/desktop-client/src/components/reports/getLiveRange.ts

```diff
@@ -0,0 +1,44 @@
+import * as monthUtils from 'loot-core/src/shared/months';
+
+import { ReportOptions } from './ReportOptions';
+import { getSpecificRange, validateRange } from './reportRanges';
+
+export function getLiveRange(cond: string, earliestTransaction: string) {
+  let dateStart;
+  let dateEnd;
+  const rangeName = ReportOptions.dateRangeMap.get(cond);
+  switch (rangeName) {
+    case 'yearToDate':
+      [dateStart, dateEnd] = validateRange(
+        earliestTransaction,
+        monthUtils.getYearStart(monthUtils.currentMonth()) + '-01',
+        monthUtils.currentDay(),
+      );
+      break;
+    case 'lastYear':
+      [dateStart, dateEnd] = validateRange(
+        earliestTransaction,
+        monthUtils.getYearStart(
+          monthUtils.prevYear(monthUtils.currentMonth()),
+        ) + '-01',
+        monthUtils.getYearEnd(monthUtils.prevYear(monthUtils.currentDate())) +
+          '-31',
```

Comment:
Does this need the same fix as the other bug?

Reply:
No, not for year end. Year end is always December 31.